carbon-design-system/carbon
# fix(sketch): remove autoapplied gray borders from sketch package

Closes #6922

This PR prevents the Sketch/DOM JS package from applying border styles to the icon symbols when the Carbon sketch plugin runs sync commands

#### Changelog

**Changed**

- clear default border styles from Sketch JS package

**Removed**

- unused border style resets

#### Testing / Reviewing

1. Run icon sync command(s) in Sketch plugin (updated build attached below)
2. Run the sync command again (or a different sync command)
3. Observe that no gray border styles are added to every icon symbol

[carbon-elements.sketchplugin.zip](https://github.com/carbon-design-system/carbon/files/5668960/carbon-elements.sketchplugin.zip)
